### ISSUE TITLE
In inference,predict 100 same boxes when model.eval(),but remove model.eval() ,predict 100 different boxes.

### ISSUE THREAD

**Issue by zhangyuge10** [facebookresearch/detr#375]
Thanks for contribution！This job is great！I have a little confusion .
       In my dataset,first training,and then inference，In inference,predict 100 same boxes when model.eval(),but remove model.eval() ,predict 100 different boxes.
       later，I find that dropout maybe cause.
       It make me confuse.
       Thank you for your help！

**Comment by ghy0324** [comment]
Hi! I met the same problem. Have you solved it?

**Comment by zhangyuge10** [comment]
> Hi! @zhangyuge10
> I met the same problem. Have you solved it?
re-downloaded the code and re-train

**Comment by zhangyuge10** [comment]
@ghy0324 ，re-downloaded the code and re-train

**Comment by mariiak2021** [comment]
Hi @ghy0324 , @zhangyuge10 I have the same problem using custom dataset. Did you understand what was the cause and how to solve the problem?
